tests/qa_automation/xfstests_prepare_issue_case.pm

```diff
@@ -0,0 +1,46 @@
+# Copyright © 2017 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary:  xfstests testsuite
+# Use the latest xfstests testsuite from upstream to make file system test
+# Maintainer: Yong Sun <yosun@suse.com>
+
+use base "xfstests_install";
+use base "xfstests_device";
+use strict;
+use testapi;
+
+sub run {
+    my $self = shift;
+    script_run("mkdir /tmp/issue_case");
+    script_run("mkdir /tmp/issue_case/generic");
+    script_run("mkdir /tmp/issue_case/shared");
+    script_run("mkdir /tmp/issue_case/xfs");
+    script_run("mkdir /tmp/issue_case/btrfs");
+    script_run("mkdir /tmp/issue_case/ext4");
```

Comment:
If all subdirectories need to be created, how about:
`mkdir -p /tmp/issue_case/{generic,shared,xfs,btrfs,ext4}`